guides/common/modules/con_api-overview.adoc

```diff
@@ -2,15 +2,15 @@
 = {Project} API overview
 
 You can write custom scripts and external applications that access the {Project} API over HTTP with the Representational State Transfer (REST) API provided by {ProjectServer}.
-Use the REST API to integrate with enterprise IT systems and third-party applications, perform automated maintenance or error checking tasks, and automate repetitive tasks with scripts. 
+Use the REST API to integrate with enterprise IT systems and third-party applications, perform automated maintenance or error checking tasks, and automate repetitive tasks with scripts.
```

Comment:
This line doesn't change the capitalization. Did you include this intentionally?

Reply:
No, that's just the 'Remove white space" setting in VSCode. Some of my projects require it to be enabled.